web-common/src/features/sources/modal/LocalSourceUpload.svelte

```diff
@@ -134,6 +114,9 @@
           filePath
         );
       }
+
+      // Navigate to source page
+      goto(`/source/${tableName}`);
```

Comment:
Having an issue here would block user from reuploading another file right? Is that the UX we are going for?

Reply:
Well, a couple things:
- For this PR, the desired UX is to send the user to the Source YAML editor on error
- In a future PR, we need to make distinct actions for "upload file" (add to `data/`) and "create source" (create the Source YAML file)